worker/instancepoller/aggregate_test.go

```diff
@@ -133,18 +129,29 @@ func (s *aggregateSuite) TestSingleRequest(c *gc.C) {
 	c.Assert(ids, gc.DeepEquals, []instance.Id{"foo"})
 }
 
+type credentialAPIForTest struct{}
+
+func (*credentialAPIForTest) InvalidateModelCredential(reason string) error {
+	return nil
```

Comment:
Should we record the method call and reason in a test.Stub so we can add checks to the unit tests?

Reply:
No, it is not necessary at this layer. Nothing here will call it.